src/requestReader/RequestReader.ts

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
The `RequestHeader` interface allows decoupling the library from Express.js and use it with any framework.